Sources_v3/StreamChatUI/ChatChannel/MessageComposer/MessageComposerViewController.swift

```diff
@@ -207,15 +214,42 @@ open class MessageComposerViewController<ExtraData: ExtraDataTypes>: ViewControl
             
             messageController?.createNewReply(
                 text: text,
+                attachments: attachmentSeeds,
                 showReplyInChannel: composerView.checkmarkControl.isSelected
             )
         } else {
-            controller?.createNewMessage(text: text)
+            controller?.createNewMessage(text: text, attachments: attachmentSeeds)
         }
     }
     
-    @objc func showImagePicker() {
-        present(imagePicker, animated: true, completion: nil)
+    @objc func showAttachmentsPicker() {
+        var actionSheet: UIAlertController {
+            let actionSheet = UIAlertController(title: nil, message: "Choose attachment type: ", preferredStyle: .actionSheet)
+            
+            actionSheet.addAction(UIAlertAction(title: "File", style: .default, handler: { [weak self] _ in
+                guard let self = self else { return }
+                self.present(self.documentPicker, animated: true, completion: nil)
+            }))
+            
+            actionSheet.addAction(UIAlertAction(title: "Photo", style: .default, handler: { [weak self] _ in
+                guard let self = self else { return }
+                self.present(self.imagePicker, animated: true, completion: nil)
+            }))
+            
+            actionSheet.addAction(UIAlertAction(title: "Cancel", style: .cancel))
+            
+            return actionSheet
+        }
```

Comment:
nit: I know this is just a temp solution but we should still use the string file for this